File: android/src/main/kotlin/com/pagecall/flutter_pagecall/PagecallFlutterPlugin.kt
package com.pagecall.flutter_pagecall

import androidx.annotation.NonNull
import io.flutter.embedding.engine.plugins.FlutterPlugin

class PagecallFlutterPlugin : FlutterPlugin {

    override fun onAttachedToEngine(@NonNull flutterPluginBinding: FlutterPlugin.FlutterPluginBinding) {
        val messenger = flutterPluginBinding.binaryMessenger
        val registry = flutterPluginBinding.platformViewRegistry

        registry.registerViewFactory(
            "com.pagecall/flutter_pagecall",
            FlutterPagecallViewFactory(messenger)
        )
    }

    override fun onDetachedFromEngine(@NonNull binding: FlutterPlugin.FlutterPluginBinding) {
//        channel.setMethodCallHandler(null)
    }
}

File: ios/Classes/PagecallFlutterPlugin.swift
import Flutter
import Pagecall

public class PagecallFlutterPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
      let messenger = registrar.messenger()
      let factory = FlutterPagecallViewFactory(messenger: messenger)

      registrar.register(factory, withId: "com.pagecall/flutter_pagecall")

      PagecallWebView.configure()
  }
}
